pkgs/development/python-modules/pbr/default.nix

```diff
@@ -11,13 +16,19 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ setuptools ];
 
-  # circular dependencies with fixtures
+  # check in passthru.tests.pytest to escape infinite recursion with fixtures
   doCheck = false;
```

Comment:
Replace `doCheck = false;` with:
```
  installCheckPhase = ''
    $out/bin/pbr --version | grep ${version} > /dev/null
  '';
```
It overwrites checkPhase + tests for version.

Reply:
We already run the tests in passthru and have an import check. This does not really add anything and usually is only used if there are not tests and you can't do an import check because it is a for example a go binary.

Reply:
I understand this is not critical. It was just a suggestion (as always is).
As I understood, `import check` only checks for module not CLI application.
This works in Python too. I have tested it.
I found it useful in cases where:
* Package hash hasn't been properly updated. It triggers.
* Upstream has messed up versions. (As happened here to a package I maintain: https://github.com/temporalio/temporal/pull/1744)
* This is the manual test I do when testing packages. I build, check version.
Should I stop adding this to every CLI application I package?